pkg/mimirpb/symbols.go

```diff
@@ -68,6 +68,7 @@ func reuseSymbolsSlice(s []string) {
 	for i := range s {
 		s[i] = ""
 	}
+	//nolint:staticcheck
```

Comment:
Could you please add more explanation about why you're disabling the linter here? And is it possible to scope this just to a particular rule, rather than staticcheck as a whole?

Reply:
I'll try.

Reply:
AFAICT, it's not possible to inline disable a specific staticcheck rule. Grepping the code base backs this up. I added an explanation.